### PR TITLE
Use WGS84 on interop publishers under mission_info and obstacles

### PR DESCRIPTION
**Pull request by yas1234**
Addressing #19. I will have to figure out how to migrate the bags still.

### CODE REVIEW

File: msg/GeoCylinder.msg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+geographic_msgs/GeoPoint center # The center of the cylinder, altitude is disregarded.
+float64 height # Height of the cylinder in m.
+float64 radius # radius od the cylinder in m.
```
Comment (anassinator) [inline msg/GeoCylinder.msg]: s/od/of

File: msg/GeoObjectArray.msg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# There might be a better way to do this msg
```
Comment (anassinator) [inline msg/GeoObjectArray.msg]: Yeah let's not do this.
Have separate `GeoCylinderArrayStamped` messages and so on.
Also we don't need the underlying `GeoCylinder` messages to be individually stamped.

File: msg/GeoSphere.msg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+geographic_msgs/GeoPoint center # Center of the sphere.
```
Comment (anassinator) [inline msg/GeoSphere.msg]: We need to be clear about whether we are relative to sea level or something

Comment (yas1234) [inline msg/GeoSphere.msg]: This should be a standard for the interop package itself, not just case by case imo.

File: msg/GeoCylinder.msg (hunk 1 above)
Comment (yas1234) [inline msg/GeoCylinder.msg]: fixed the "od"-> "of". no idea what the other issues are.

File: msg/GeoSphere.msg (hunk 3 above)
Comment (anassinator) [inline msg/GeoSphere.msg]: Yeah but I just mean which one is used should be clear in the comment of each message

File: CMakeLists.txt [hunk 4]
```diff
@@ -18,11 +19,18 @@ add_message_files(FILES
...
+  GeoCylinderArrayStamped.msg
+  GeoPolygon.msg
+  GeoPolygonStamped.msg
+  GeoSphere.msg
+  GeoSphereArrayStamped.msg
   Orientation.msg
   Shape.msg
   Target.msg
   TargetType.msg
   UTMZone.msg
```
Comment (anassinator) [inline CMakeLists.txt]: can you remove all the unuzed stuff as well

File: scripts/obstacles_client.py [hunk 5]
```diff
@@ -7,6 +7,7 @@
 from simplejson import JSONDecodeError
 from interop import InteroperabilityClient
 from visualization_msgs.msg import MarkerArray
+from interop.msg import GeoObjectArray
```
Comment (anassinator) [inline scripts/obstacles_client.py]: i thought this doesn't exist anymore

**Comment by anassinator** [comment]
@yas1234 I think this is still broken, especially `obstaces_client.py`. Can you fix and test please?
@foochuanwei Can you take a look too please?

**Comment by yas1234** [comment]
Sorry I've been a bit busy this week, I'll take a look today.

**Comment by yas1234** [comment]
It works now

File: CMakeLists.txt (hunk 4 above)
Comment (yas1234) [inline CMakeLists.txt]: hm? i've already removed the unused stuff

**Comment by anassinator** [comment]
Looks good, but can you remove `UTMZone` before merging since it's no longer needed?

**Comment by yas1234** [comment]
Okay no more `UTM`

**Comment by anassinator** [comment]
oh and the README sorry :P

**Comment by yas1234** [comment]
Okay I updated the readme as well

**Pull request merged**